spec/requests/users_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'users controller', type: :request do
+  describe 'GET user routes' do
+    describe 'GET /users' do
+      before(:example) { get '/users' }
+
+      it 'should return 200 http_status_code' do
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'should render the correct template' do
+        expect(response).to render_template(:index)
+      end
+    end
+
+    describe 'GET /users/:id' do
+      before(:example) { get '/users/1' }
+
+      it 'should return 200 http_status_code' do
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'should render the correct template' do
+        expect(response).to render_template(:show)
+      end
+    end
```

Comment:
As required, You are missing out on tests that ensure the response body includes the correct placeholder text.
`Example : expect(response.body).to include("Mindly Users")`

![image](https://user-images.githubusercontent.com/66112760/154549232-269d8d4a-6af0-43f3-993f-f22d36bd6bc9.png)

Kindly add this test action for both `index` and `show`🙂